siliconcompiler/core.py

```diff
@@ -4153,9 +4153,10 @@ def run(self):
                 self.error('Cannot pass "-step" parameter into remote flow. A steplist including '
                            'an "import" task and at least one EDA task is required.',
                            fatal=True)
+            cur_steplist = self.get('option', 'steplist')
             pre_remote_steplist = {
-                'steplist': self.get('option', 'steplist'),
-                'set': self.get('option', 'steplist', field='set'),
+                'steplist': cur_steplist,
+                'set': self.get('option', 'steplist', field='set') if cur_steplist else False,
```

Comment:
The "set" field no longer exists as of https://github.com/siliconcompiler/siliconcompiler/pull/1311 (based on some feedback to make the handling of values more consistent). I missed this when refactoring though, my bad.

For a replacement, it's probably best to use the `Schema._is_set(...)` helper function. That takes in a cfg dictionary right now, but I think it would make sense to add a wrapper around it to take in a keypath, call `_search()` to grab the dict, and pass that in.

Reply:
Okay, I'll give that a try.